Remove deprecated dependency: Textrue's Embeddium Options

HeroOfFire45 opened the pull request:
Embeddium 0.3.16 adds similar functionality to TEO, and as such the mod is discontinued and Embeddium no longer supports loading it. As such, Embeddium/Magnesium Extras no longer works as is. (See issues #63 and #67)

The modification here allows it to still work; all it does is remove the deprecated dependency, and the Embeddium dependency allows it to still run without issues. The Embeddium dependency is also updated to require version 0.3.16 or later alongside this change, as it will break on prior versions due to not having TEO-like functionality.

HeroOfFire45 committed:
Remove deprecated dependency: Textrue's Embeddium Options

Embeddium 3.16+ adds similar functionality to TEO, and as such the mod is discontinued and Embeddium no longer supports loading it. As such, Embeddium/Magnesium Extras no longer works as is.

The modification here allows it to still work; all it does is remove the deprecated dependency. The Embeddium dependency should be updated to require version 3.16+ alongside this change.

HeroOfFire45 commented on src/main/resources/META-INF/mods.toml:
```diff
@@ -35,10 +35,3 @@ displayTest="IGNORE_SERVER_VERSION"
     versionRange="${embeddiumrange}"
```
Need to change range to be version 0.3.16 and up

Cephelo commented:
Might also need to remove its reference in the build.gradle file, not sure though.  Hopefully this gets looked at soon!

HeroOfFire45 committed:
remove TEO from build.gradle

HeroOfFire45 commented:
> You may also need to remove its reference in the build.gradle file, I'm not sure though. Hopefully this gets looked at soon!

Done. Mod seems pretty inactive, so not keeping my hopes up - but this will save them some work if they come back to it.

HeroOfFire45 committed:
Remove TEO dependency from gradle.properites and update Embeddium dependency

Mod will now require Embeddium 0.3.16+ since it has the same functionality as TEO for previous versions.

Ali811G commented:
how do i download this version?

HeroOfFire45 commented:
I can't compile a .jar since that'd be infringing on the original creators'
rights to this mod, but you can open the latest .jar using 7zip or similar
and make the same edits using your favorite text/code editor

On Sun, Aug 4, 2024 at 11:31 AM Ali811 ***@***.***> wrote:

> how do i download this version?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/anthxnymc/MagnesiumExtras/pull/66#issuecomment-2267613648>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACZBSOZPX7XYN2IE6NJLOV3ZPZQOJAVCNFSM6AAAAABKN7AVWSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDENRXGYYTGNRUHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
